### PR TITLE
doc: Adjust commit message template for the guix.sigs repo

### PR DESCRIPTION
**Pull request by maflcko**
Seems to be the most common template used, so adjust this here, too.

### CODE REVIEW

**Review by hebasto — APPROVED** [review]
ACK fa3b92d20570c72a3178c693afb81e105849805f.

Curly braces could be skipped for clarity.

**Comment by MarcoFalke** [comment]
I like to always add them for clarity

File: doc/release-process.md [hunk 1]
```diff
@@ -136,7 +136,7 @@ Commit your signature to guix.sigs:
 ```sh
 pushd ./guix.sigs
 git add "${VERSION}/${SIGNER}"/noncodesigned.SHA256SUMS{,.asc}
-git commit -m "Add ${VERSION} unsigned sigs for ${SIGNER}"
+git commit -m "Add ${SIGNER} Guix attestations for ${VERSION} non-codesigned"
```
Comment (laanwj) [inline doc/release-process.md:139]: Don't think this needs to mention "Guix", it's the guix signatures repository so that is implicit.

Comment (MarcoFalke) [reply doc/release-process.md:139]: I am also thinking if the sentence was proper English before.

Shouldn't it say "Add attestations *by* $SIGNER for $VERSION non-codesigned"?

**Review by hebasto — APPROVED** [review]
re-ACK fafade9c79f55c186c1938ce3e27077d12dee6c5

File: doc/release-process.md (hunk 1 above)
Comment (laanwj) [reply doc/release-process.md]: Yes, that'd be even better I think.

**Comment by laanwj** [comment]
ACK fafade9c79f55c186c1938ce3e27077d12dee6c5

**Pull request merged**